### DISCUSSION TITLE
upcomming features list

### DISCUSSION THREAD

**Discussion by onno-timmerman** [wagtail/wagtail-ai#121] [Q&A] [answered]
Is there somewhere an planned features list so I don't make redundand feature request.

**Comment by thibaudcolas** [comment, marked as answer]
yep! [AI in the CMS: steering the ecosystem](https://wagtail.org/blog/ai-in-the-cms-steering-the-ecosystem/). Redundant feature requests are welcome, we can always do a bit of triage after. Here are the specific areas we are interested in. Note some might warrant a separate package:

- **Content summaries**: for example page descriptions, or even titles.
- **Image descriptions**: captions, alt text, ideally contextual (adjusted for surrounding content).
- **Content classification**: auto-filling categories or tags.
- **Related content suggestions**: encouraging linking to existing pages where suitable.
- **Translations**: making it simpler to create multilingual sites or even one-off pages.
- **Qualitative content feedback**: think readability improvements or style guide alignment.
- **Quantitative feedback**: repeatable metrics based on the page’s content.
- **Improvements suggestions**: based on automated accessibility and SEO checks.
- **PDF to HTML conversion**: a common requirement to make content more accessible.
